spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestRuntimeFiltering.java

```diff
@@ -177,13 +177,21 @@ public void testMultipleRuntimeFilters() throws NoSuchTableException {
         "SELECT f.* FROM %s f JOIN dim d ON f.id = d.id AND f.data = d.data AND d.date = DATE '1970-01-02'",
         tableName);
 
+    String caseInsensitiveQuery = String.format(
+        "SELECT f.* FROM %s f JOIN dim d ON f.Id = d.id AND f.Data = d.data AND d.date = DATE '1970-01-02'",
```

Comment:
Can you please go all out in making this case insensitivity check a bit more clear?

Namely, finding the characters that were different from the above wasn't that quickly obvious.

I would say go wild and make as much of it as you can have funky non-standard casing. Like `select f.* from %s F join dim d ON f.Id = d.iD and f.DaTa = d.dAtA and d.dAtE = date '1970-01--02'`